src/contracts/event_factory.js

```diff
@@ -1,9 +1,16 @@
 import Config from '../../config/config';
 import Contracts from '../../config/contracts';
+import utils from '../modules/qweb3/src/utils';
 
 const Qweb3 = require('../modules/qweb3/index');
 const qweb3 = new Qweb3(Config.QTUM_RPC_ADDRESS);
 
+/** @type {number} Length of string of eventName */
+const EVENTNAME_STR_LENGTH = 32 * 2; // 32 bytes times 2 hex character
```

Comment:
We need to leave this as 32. The `Utils.stringArrayToHex` will take care of changing this to the 64 char limit.

Reply:
Okay, grepped stringArrayToHex in qweb3. Gotcha